src/internal/connector/sharepoint/data_collections.go

```diff
@@ -6,10 +6,9 @@ import (
 
 	"github.com/alcionai/clues"
 
-	dapi "github.com/alcionai/corso/src/internal/connector/discovery/api"
 	"github.com/alcionai/corso/src/internal/connector/graph"
 	"github.com/alcionai/corso/src/internal/connector/onedrive"
-	"github.com/alcionai/corso/src/internal/connector/sharepoint/api"
+	sapi "github.com/alcionai/corso/src/internal/connector/sharepoint/api"
```

Comment:
We try not to alias the import that is most closely related to the current package.  Since this is a test suite for the sharepoint package, that means we would keep this api import unalilased.

Reply:
Good rule of thumb 👍